<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="mode_1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>180</width>
      <height>116</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>5</y>
      <width>180</width>
      <height>116</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_3">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>5</y>
      <width>180</width>
      <height>116</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>5</y>
      <width>180</width>
      <height>116</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg_mode_1">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>43</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mode_1">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>18</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="split_1">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>18</y>
      <width>1</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="split_2">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>18</y>
      <width>1</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="split_3">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>18</y>
      <width>1</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mode_2">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>18</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mode_3">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>18</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mode_4">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>18</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bg_mode_2">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>43</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg_mode_3">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>43</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg_mode_4">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>43</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
